app/main.py

```diff
@@ -16,4 +16,14 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+types = (lucky_number, pi, one_is_a_prime_number, name,
+         my_favourite_films, profile_info, marks, collection_of_coins)
+sorted_variables = {"mutable": [], "immutable": []}
+
+for obj in types:
+    if isinstance(obj, set) or isinstance(obj, dict) or isinstance(obj, list):
+        sorted_variables["mutable"].append(obj)
+    else:
+        sorted_variables["immutable"].append(obj)
+
+print(sorted_variables)
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.

Reply:
Sorry, but I can`t understand what are you trying to explain me. I need to do it manually in code without for?

Reply:
> No additional logic is required - just pass the variables to the correct list in the dictionary.
Sorry, but I can`t understand what are you trying to explain me. I need to do it manually in code without for?